packages/account/src/Sections/Verification/ProofOfOwnership/proof-of-ownership-form.jsx

```diff
@@ -94,8 +119,22 @@ const ProofOfOwnershipForm = ({ cards, updateAccountStatus }) => {
                         files_to_send.documentType = 'proof_of_ownership';
                         const response = await uploader.upload(files_to_send);
                         if (response.warning) {
-                            // eslint-disable-next-line no-console
-                            console.warn(response);
+                            if (response.warning.trim() === 'DuplicateUpload') {
+                                form_ref.current.errors.data = document_upload_errors;
+                                if (typeof form_ref.current.errors.data[index] !== 'object') {
+                                    form_ref.current.errors.data[index] = {};
```

Comment:
please destructure form_ref.current.errors to prevent code duplications

Reply:
@ali-hosseini-fs, If I destructure that array, then I'll be working with a different array altogether, yet I want to modify the array referenced by form_ref, because form_ref is a use ref instance. I want to add errors directly to it, so that the errors are displayed in the corresponding places on the form.

Reply:
Hello @ali-hosseini-fs ,

This is done. 

Thank you